src/pages/Login.tsx

```diff
@@ -80,14 +80,6 @@ const Login: React.FC = () => {
                             </Button>
                         </form>
                     </CardActions>
-                    <CardContent>
-                        <Typography variant="caption" color="initial">
-                            <FormattedMessage id="login.help.message" />
-                            <NavLink to="/login/help">
-                                <FormattedMessage id="cta.clickHere" />
-                            </NavLink>
-                        </Typography>
-                    </CardContent>
```

Comment:
We probably won't ever have a "help logging in" since we'll be using OpenID or something like that.